### DISCUSSION TITLE
Electricity transmission settings about AC, DC and VSC

### DISCUSSION THREAD

**Discussion by narata2019** [NREL/ReEDS-2.0#122] [Q&A: Features]
Hello, ReEDS team！

I want to ask if there is transmission direction difference among theses 3 type lines in ReEDS ? It seems all there types lines can operate in both directions.

As far as I know, in China LCC-HVDC ( as DC in ReEDS)  can only operate in one direction, such as from Northwset China to Shanghai. It can not transmit electricity in the opposite direction. While AC and VSC-HVDC can transmit electricity in both directions.

Many thanks!
Siming

**Comment by patrickbrown4** [comment]
All three transmission types are assumed to be able to operate bidirectionally in ReEDS. Switching the direction of flow is slower and more difficult for LCC since it requires reversing the voltage polarity, but the usual assumption is that it can still be done. Most LCC lines are built to link large generators to distant load centers, so in practice they do tend to operate in just one direction, but we assume they can operate bidirectionally in all of our studies (see this study as an example: https://doi.org/10.1109/TPWRS.2021.3115092).

**Comment by narata2019** [comment]
Many thanks! Your detailed answer helps a lot!
